docs/CONTRIBUTING.md

```diff
@@ -79,7 +81,7 @@ pnpm astro:dev
 
 ## Some Guidelines to follow
 
-Make sure you have fulfilled the requirements and local development steps mentioned [here](../README.md#local-development-🧑🏻‍💻).
```

Comment:
This link was broken. Was it meant to link to the Local Development section in this same doc, or something else?

Reply:
Yes exactly.